homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/js/homework.js

```diff
@@ -0,0 +1,48 @@
+function calculate(firstValue, secondValue, operation) {
+    let result = 0;
+    const first = Number(firstValue);
+    const second = Number(secondValue);
+    const resultHeaders = document.getElementById('result_headers');
+
+    if (Number.isNaN(first) || Number.isNaN(second)) {
+        resultHeaders.innerHTML = '<h1>Enter a number</h1>';
+        return 'Enter a number';
+    }
+
+    if (first === '' || second === '') {
```

Comment:
For the moment of this check, you've converted the values to Number and got 0
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/66c0fa55-594f-49dd-a39b-33cfad0f1651)

Reply:
And add the word `games` here

Reply:
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/c4d3a50a-b206-4572-95e6-18a20d1056af)

Reply:
> For the moment of this check, you've converted the values to Number and got 0 ![image](https://user-images.githubusercontent.com/98031768/280645435-66c0fa55-594f-49dd-a39b-33cfad0f1651.png)

Could you please tell me what should be instead of a zero?

Reply:
you should check the `firstValue` not the `first`